### PR TITLE
Add scroll to top in scroll window #342

### LINKED ISSUE #342: Add scroll to top in scroll window
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/susi_server

**Current behavior:**
Only scroll to bottom present as of now

**Expected behavior:**
Scroll to top to see previous chats

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #342 

Closed PR https://github.com/fossasia/susi_chromebot/pull/371 due to some multiple merge conflicts.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Added scroll to top functionality
scroll-up-down

#### Changes proposed in this pull request:
 - Scroll to top
 - Code indentation
 - Removed redundant new lines

### CODE REVIEW

**Comment by fragm3** [comment]
Using ternary wont work here, error:
screenshot 2018-10-06 at 4 14 46 pm

@ms10398 @gabru-md @rupav @zamhaq please review

**Comment by gabru-md** [comment]
@fragm3 can you try it in this way ?

```js
scrollIconBottom.style.display = (messages.scrollTop === 0)? "block":"none" ;
```

the screenshot says that there is no assignment, so you can use the way I suggested above. Give it a try and if it works, then it is good, else we can simply merge this PR of yours and then iterate on this one.

**Comment by gabru-md** [comment]
helpss

something like this should get this working in no time. if there is still some problem then we can do it in the later PRs.

Thanksa

**Comment by fragm3** [comment]
Done , please review @gabru-md @ms10398 @zamhaq

File: src/style.css [hunk 1]
```diff
@@ -52,18 +52,18 @@ body {
...
-     padding: 0px 8px;
-     border-radius: 50%;
-     padding-top: 5px;
-     display: none;
- }
+.scroll-icon-bottom, .scroll-icon-top{
+    position: absolute;
+    right: 20px;
+    cursor: pointer;
+    z-index: 99;
```
Comment (rupav) [inline src/style.css]: Just remove this line, and squash your commits. z-index is already being added inline (and set to 1).

**Comment by fragm3** [comment]
@rupav Done the changes, squashed the commits.

**Pull request merged**